python/py-cbor/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-cbor
+version             1.0.0
+platforms           darwin
+license             Apache
+maintainers         {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
+
+description         RFC 7049 Concise Binary Object Representation
+long_description    ${description}
+
+homepage            https://bitbucket.org/bodhisnarkva/cbor
+master_sites        pypi:c/cbor
+distname            cbor-${version}
```

Comment:
I think pypi2port still adds these, but doesn't the python portgroup handle setting `master_sites`, `distname` and maybe even `livecheck.type pypi` for you now, so they shouldn't be set in the port anymore?

Reply:
You're right. I edited `pypi2port` to default to Python 3.8 just now, so perhaps I can change it to no longer add `master_sites`, `distname` and the `livecheck` block...